fleetspeak/src/client/generic/proto/fleetspeak_client_generic/config.proto

```diff
@@ -30,6 +30,11 @@ message Config {
   // The format is a URL.
   // See https://golang.org/pkg/net/http/#Transport.Proxy for details.
   string proxy = 7;
+
+  // If set, the server will validate the client certificate from the request header.
+  // This should be used if TLS is terminated at the load balancer and client certificates
+  // can be passed upstream to the fleetspeak server as an http header.
```

Comment:
```
// The name of the HTTP header that the client uses to pass its certificate to
// the server frontend for identification. Required only if the server
// frontend is configured to use https_header_checksum_config.
```

Reply:
If the frontend uses https_header_config, it gets the certificate from the LB, not from the client :)

Reply:
The cert is coming in a header all the way from the client which the LB passes through to the frontend.
The LB is adding the additional checksum header.